admin_scripts/os2displaypc/chromium_setup.sh

```diff
@@ -51,17 +51,17 @@ chmod +x /usr/local/bin/rotate_screen.sh &
 cat << EOF > /home/chrome/.xinitrc
 #!/bin/sh
 
-xset -dpms
+# xset -dpms
 xset s off
 xset s noblank
 
 sleep 20
 
-sed -i 's/"exited_cleanly":false/"exited_cleanly":true/' /home/chrome/.config/chromium/Default/Preferences
+# sed -i 's/"exited_cleanly":false/"exited_cleanly":true/' /home/chrome/.config/chromium/Default/Preferences
 
-sed -i 's/"exit_type":"Crashed"/"exit_type":"None"/' /home/chrome/.config/chromium/Default/Preferences
+# sed -i 's/"exit_type":"Crashed"/"exit_type":"None"/' /home/chrome/.config/chromium/Default/Preferences
 
-sed -i 's/"restore_on_startup":[0-9]/"restore_on_startup":0/' /home/chrome/.config/chromium/Default/Preferences
+# sed -i 's/"restore_on_startup":[0-9]/"restore_on_startup":0/' /home/chrome/.config/chromium/Default/Preferences
```

Comment:
(I imagine this property corresponds to the [`RestoreOnStartup` policy option](https://cloud.google.com/docs/chrome-enterprise/policies/?policy=RestoreOnStartup), so perhaps this line can just be removed altogether?)

Reply:
Indeed!